src/pages/academy/gameSimulator/subcomponents/chapterPublisher/ChapterPublisherEditor.tsx

```diff
@@ -160,17 +155,14 @@ const ChapterEditor = React.memo(({ chapterDetail, checkpointFilenames }: Chapte
           );
         })}
       <br />
-      <Button icon="play" onClick={simulateChapter}>
-        Simulate Chapter
-      </Button>
-      <br />
+      <hr />
       <br />
       <Switch
         checked={isPublished}
         labelElement={'Published'}
         onChange={() => setIsPublished(!isPublished)}
       />
-      <Button onClick={saveChapter}>Save Changes</Button>
+      <Button onClick={saveChapter}>Save Changes</Button>{' '}
```

Comment:
Was this change intentional?

Reply:
The change was intentional to have some whitespace between two buttons - though on hindsight I'll revert this for now and look at it a bit more systematically in the UI update.

Do we prefer directly changing margins over using `{' '}` for quick whitespacing?

Reply:
I don't mind using `{' '}` – I think it's used in a couple places as well.

I was just making sure since the change appeared out of nowhere; sure, we can leave it to the UI update, thanks!